app/views/templates/MainTemplateCUI.scala.html

```diff
@@ -15,41 +15,28 @@
  *@
 
 @import config.AppConfig
-@import uk.gov.hmrc.play.views.html.layouts
 @import uk.gov.hmrc.webchat.client.WebChatClient
+@import views.html.includes.cuiSidebar
 
 @this(
     govuk_wrapper: GovukWrapperCUI,
-    sidebar: Sidebar,
-    article: Article,
     appConfig: AppConfig,
     webChatClient: WebChatClient
 )
 
-@(title: String,
-    navTitle: Option[String] = None,
-    sidebarLinks: Option[Html] = None,
-    contentHeader: Option[Html] = None,
-    bodyClasses: Option[String] = None,
-    mainClass: Option[String] = Some("cui-main"))(mainContent: Html)(implicit request : Request[_], messages: Messages)
-
-@serviceInfoContent = {}
+@(title: String, navTitle: Option[String] = Some("Ask HMRC"))(mainContent: Html)(implicit request : Request[_], messages: Messages)
 
 @sidebarContent = {
-    @if(sidebarLinks.isDefined) {
-        @sidebar(sidebarLinks.get, Some("sidebar"))
-    }
-}
-
-@wrapMainContent = {
-    @mainContent
+    @cuiSidebar()
 }
 
 @scriptElem = {
 
 <script>
     window.isCUI = true;
 </script>
+<!--This is only used for fading elements.-->
+<script src='@routes.Assets.versioned("javascripts/thirdparty/jquery-3.5.1.min.js")' type="text/javascript"></script>
```

Comment:
I would seriously consider adding in another fading library or rolling your own rather than jQuery purely for this function.